apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -41,6 +42,67 @@ export class CommandLineConfiguration {
     return new CommandLineConfiguration(commandLineJson);
   }
 
+  /**
+   * Loads the configuration from the specified file and applies any ommited default build
+   * settings.  If the file does not exist, then an empty default instance is returned.
+   * If the file contains errors, then an exception is thrown.
+   */
+  public static loadFromFileAndDefault(jsonFilename: string): CommandLineConfiguration {
+    let commandLineJson: ICommandLineJson | undefined = undefined;
+    if (FileSystem.exists(jsonFilename)) {
+      commandLineJson = JsonFile.load(jsonFilename);
+
+      // TODO: Investigate
+      const defaultBuildSettings: CommandJson[] =
```

Comment:
Shouldn't these defaults also apply if the jsonFile doesn't exist?

Reply:
No, from my understanding defaults are applied if the command-line config file doesn't exist but it happens at [_addDefaultBuildActions()](https://github.com/microsoft/rushstack/blob/fd7859b96485a097855efc6445f495784b3becbb/apps/rush-lib/src/cli/RushCommandLineParser.ts#L193). This function checks if the build or rebuild actions exists and adds the actions if necessary.